src/installer/tests/Assets/TestUtils/TestProjects.targets

```diff
@@ -3,29 +3,9 @@
     This file is imported by the test projects from the artifacts dir or the src/tests dir. It
     provides basic info needed for restore and build with the vanilla SDK.
   -->
-  <PropertyGroup>
-    <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion>
-  </PropertyGroup>
+
   <!--
     We are using to best emulate the live version. This should be removed once we start using the live bits.
```

Comment:
Can we remove this comment now?

Can the whole file be removed?

Reply:
Nope, that issue is still accurate, we need to use the live built targeting pack to build the installer tests. Using a prebuilt here seems unnecessary and just adds a prebuilt dependency to our build.

Reply:
I don't understand. This file only contains comments. What value is it serving?

Reply:
installer tests don't import the rest of the repo's infrastructure. That file is the point where properties or targets should live which are necessary for the installer tests. That file was already exactly in the current state (empty) in 5.0 before the KnownFrameworkReference was added, just without the comment.